portsyncd/linksync.cpp

```diff
@@ -165,16 +165,18 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
     struct rtnl_link *link = (struct rtnl_link *)obj;
     string key = rtnl_link_get_name(link);
 
+    vector<FieldValueTuple> temp;
     if (key.compare(0, INTFS_PREFIX.length(), INTFS_PREFIX) &&
         key.compare(0, LAG_PREFIX.length(), LAG_PREFIX) &&
-        key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX))
+        key.compare(0, MGMT_PREFIX.length(), MGMT_PREFIX) &&
+        !m_portTable.get(key, temp))
     {
         return;
     }
 
     unsigned int flags = rtnl_link_get_flags(link);
     bool admin = flags & IFF_UP;
-    bool oper = flags & IFF_LOWER_UP;
+    bool oper = flags & IFF_RUNNING;
```

Comment:
Remove this change from this PR. Raise separate issue on sonic-buildimage fix this as part of the Issue.

Reply:
Fixed

Reply:
Issue https://github.com/Azure/sonic-buildimage/issues/6295 raised to request to change to IFF_RUNNING and PR Azure/sonic-swss#1568 has the changes